apps/degen-web-ui/src/core/interfaces/degen-service.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { PoapAdminDTO } from './poap-admin.dto';
+import { PoapSettingsDTO } from './poap-settings.dto';
+
+export interface PoapService {
```

Comment:
you don't need interfaces for services like this

Reply:
oh yeah meant to delete this bad boy